pkg/kn/commands/service/service_update.go

```diff
@@ -55,19 +56,19 @@ func NewServiceUpdateCommand(p *commands.KnParams) *cobra.Command {
 			}
 			service = service.DeepCopy()
 
-			err = editFlags.Apply(service, cmd)
-			if err != nil {
+			if err = editFlags.Apply(service, cmd); err != nil {
```

Comment:
dont think we are switching to the collapsed style judging by the rest of the code.

Reply:
fair point, shouldn't touch existing code for good (aka better) reasons.